rdwatch/core/migrations/0036_modelrunupload_alter_animationmodelrunexport_user_and_more.py

```diff
@@ -56,4 +59,22 @@ class Migration(migrations.Migration):
                 ),
             ],
         ),
+        migrations.AlterField(
+            model_name='animationmodelrunexport',
+            name='user',
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name='core_animationmodelrunexports',
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AlterField(
+            model_name='animationsiteexport',
+            name='user',
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name='core_animationsiteexports',
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
```

Comment:
Shouldn't this be in a new migration, instead of modifying an existing one? Otherwise, this won't apply properly to deployments that have already has migration 36 applied.

Reply:
Good catch, I think it got it merged into the updates for the modelrunupload.  I'll make it it's own migration.